### ISSUE TITLE
Scaled loss / magic number

### ISSUE THREAD

**Issue by apisarek** [Puzer/stylegan#8]
Hi! Great project :)

I would like to ask where does the scale in perceptual model loss come from. I mean that line:
`self.loss = tf.losses.mean_squared_error(self.features_weight * self.ref_img_features,
                                                    self.features_weight * generated_img_features) / 82890.0`

I tried `82890.0` to factorize it `2 * 3^3 * 5 * 307` to check if it's the size of output features or anything similar, but apparently no. Also I searched through all the Github to find exactly the same scale in other projects, but found no good match.

**Comment by zhujiapeng** [comment]
+1

**Comment by aiueogawa** [comment]
@Puzer 
+1

**Comment by arunasank** [comment]
Any luck here?

**Comment by akshaysravindran** [comment]
Confused by this as well. I could not find a good justification for this number or how the scale should change w.r.t. input or architectural changes. Any updates on this?
